src/azure-cli/azure/cli/command_modules/ams/operations/sp.py

```diff
@@ -300,10 +285,8 @@ def create_application(client, display_name, homepage, years, password,
 
 
 def update_application(client, app_object_id, display_name):
-    app_update_param = ApplicationUpdateParameters(display_name=display_name)
-
     try:
-        return client.patch(app_object_id, app_update_param)
+        return client.application_patch(app_object_id, {"displayName": display_name})
```

Comment:
I am about to rename it to `application_update`. Will let you know when I finish.

Reply:
This has been done in https://github.com/Azure/azure-cli/pull/22798